deployment/terraform/assets.go

```diff
@@ -39,6 +40,56 @@ func MustAssetString(name string) string {
 
 // RestoreAssets writes an embedded asset to the given directory
 func RestoreAssets(dir, name string) error {
+	assetPath := "assets/" + name
+
+	// Check if it's a directory
+	fileInfo, err := fs.Stat(assetsFS, assetPath)
+	if err != nil {
+		return err
+	}
+
+	if fileInfo.IsDir() {
```

Comment:
Nit: We're not checking if it's a directory with the `Stat` call, but with the `IsDir` call.